src/compute/src/memory_management/policy.rs

```diff
@@ -15,140 +15,46 @@
 use std::sync::atomic::AtomicU64;
 use std::sync::Arc;
 
-use anyhow::anyhow;
 use risingwave_batch::task::BatchManager;
-use risingwave_common::error::Result;
 use risingwave_common::util::epoch::Epoch;
-use risingwave_common::util::pretty_bytes::convert;
 use risingwave_stream::task::LocalStreamManager;
 
 use super::{MemoryControl, MemoryControlStats};
 
-/// `FixedProportionPolicy` performs memory control by limiting the memory usage of both batch and
-/// streaming to a fixed proportion.
-pub struct FixedProportionPolicy {
-    /// The proportion of streaming memory to all available memory for computing. This should
-    /// always fall in (0, 1). The proportion of batch memory will be 1
-    /// -`streaming_memory_proportion`.
-    streaming_memory_proportion: f64,
+/// `JemallocMemoryControl` is a memory control policy that uses jemalloc statistics to control. It
+/// assumes that most memory is used by streaming engine and does memory control over LRU watermark
+/// based on jemalloc statistics.
+#[derive(Debug)]
+pub struct JemallocMemoryControl {
+    threshold_graceful: usize,
+    threshold_aggressive: usize,
 }
```

Comment:
Not sure whether it's a good idea to put configuration values into a policy (sth. like concrete vs abstract).

Reply:
Emmmm I prefer this style a little bit, because these values belong to the policy's config in my mind, and besides, these values don't change in loops